e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -1189,6 +1187,41 @@ func (s *IbcEurekaTestSuite) ICS20TransferTimeoutFromEthereumToCosmosChainTest(c
 	}))
 }
 
+func (s *IbcEurekaTestSuite) createICS20MsgSendPacket(
+	denom string,
+	amount *big.Int,
+	receiver string,
+	sourceChannel string,
+	timeoutTimestamp uint64,
+	memo string,
+) ics26router.IICS26RouterMsgsMsgSendPacket {
+	msgSendTransfer := ics20transfer.IICS20TransferMsgsSendTransferMsg{
+		Denom:            denom,
+		Amount:           amount,
+		Receiver:         receiver,
+		SourceChannel:    sourceChannel,
+		DestPort:         transfertypes.PortID,
+		TimeoutTimestamp: timeoutTimestamp,
+		Memo:             memo,
+	}
+	msgSendPacket, err := s.ics20Contract.CreateMsgSendPacket(s.GetCallOpts(s.key), msgSendTransfer)
```

Comment:
Interesting use of this function. We should be able to call this without s.key right? Essentially as a query?

Reply:
It needs the correct caller because it sets sender. But I will refactor it so it doesn't do that anymore